ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -3155,6 +3155,16 @@ tests:
     test:
     - chain: cucushift-sdn-migration-ovn
     workflow: cucushift-installer-rehearse-vsphere-ipi-sdn
+- as: vsphere-ipi-static-ip-f14
+  cron: 28 5 3,18 * *
+  steps:
+    cluster_profile: vsphere-8
```

Comment:
Can we use `vpshere-connected` cluster profile here?

Reply:
Updated, thanks.